pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -96,7 +98,12 @@ func (n *nsLifecycle) syncNS(obj *v1.Namespace) (bool, error) {
 		return false, err
 	}
 
-	return hasPRTBs, nil
+	hasApps, err := n.ensureAppPrivilegesAddToNamespaces(obj)
+	if err != nil {
+		return false, err
+	}
+
+	return hasPRTBs || hasApps, nil
```

Comment:
Do we need hasApps for the result? the return value of syncNS() is used for setting initialRoleCondition of a namespace.

Reply:
If a new namespace created in a project and we have an app using an injected sa of this project, we should also create role and role bindings for it. Same logic and same meaning as we do for PRTB.